t/t1001-mf-priority.t

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+test_description='Test multi-factor priority plugin'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+# BULK_UPDATE=${FLUX_BUILD_DIR}/t/scripts/send_fake_payloads.py
+
+export TEST_UNDER_FLUX_NO_JOB_EXEC=y
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+        flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
+'
+
+test_expect_success 'list all loaded plugins' '
+        flux jobtap list -a > test.txt
+'
+
+test_expect_success 'make sure multi-factor priority plugin is loaded' '
+        test_cmp test.txt ${FLUX_BUILD_DIR}/t/expected/loaded_priority.txt
+'
```

Comment:
Suggestion: do not use `test_cmp` to test output of `flux jobtap list` because you never know, some default plugin besides just your priority plugin could be loaded. Instead, combine these two tests into one and use `grep` to ensure your plugin was loaded:

```
test_expect_success 'make sure multi-factor priority plugin is loaded' '
        flux jobtap list | grep mf_priority
'
```

Remove `loaded_priority.txt` from version control

Reply:
Ah, good point. Thank you. I've just pushed up a change in the test to just use `grep` to make sure my plugin is loaded.